File: data/src/main/java/org/oppia/data/backends/gae/model/AdditionalStorySummary.java
package org.oppia.data.backends.gae.model;

import com.squareup.moshi.Json;

public class AdditionalStorySummary {

  /*
   * Parent Hierarchy: Topic -> AdditionalStorySummary
   */

  @Json(name = "id")
  public String id;

  @Json(name = "title")
  public String title;

  @Json(name = "description")
  public String description;

}

File: data/src/main/java/org/oppia/data/backends/gae/model/AdditionalStorySummary.kt
package org.oppia.data.backends.gae.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class AdditionalStorySummary(

  @Json(name = "id") val id: String,
  @Json(name = "title") val title: String,
  @Json(name = "description") val description: String

)